### PR TITLE
jump_search

### LINKED ISSUE #163: Add jump search in JavaScript
Add jump search in JavaScript programming language .

### PR DESCRIPTION
**Pull request by Amit366**
closes : #163 

implemented jump_search using javascript

### CODE REVIEW

File: Search_Techniques/Jump_Search/jump_search.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  
+  var upperBound = Math.min(step, length);
+  while (Searcharray[lowerBound] < Searchvalue)
+  {
+    lowerBound++;
+    if (lowerBound == upperBound){
+      return -1;
+    }
+  }
+  if (Searcharray[lowerBound] == Searchvalue){
```
Comment (ankitapuri) [inline Search_Techniques/Jump_Search/jump_search.js]: Its === in JS for equality .  
Also add comments explaining your code .

**Review by ankitapuri — APPROVED** [review]

**Comment by Amit366** [comment]
@ankitapuri please add appropriate tags.

**Comment by ankitapuri** [comment]
> @ankitapuri please add appropriate tags.

I have already added in the issues section . Which tag is missing

**Comment by Amit366** [comment]
Maybe add it on the merged pr then I think it's counted

**Comment by ankitapuri** [comment]
> Maybe add it on the merged pr then I think it's counted

I am maintaining a spreadsheet . Don't need to worry

**Comment by Amit366** [comment]
@ankitapuri The points of this PR has not been considered. Please check as soon as possible

**Pull request merged**